source/tutorials/packaging-existing-software.md

```diff
@@ -8,26 +8,42 @@ myst:
 
-In the long term, Nix helps tremendously in alleviating that stress, but when *first* packaging existing software with Nix, it's common to encounter missing dependencies preventing builds from succeeding.
+In the long term, Nix helps tremendously with alleviating such problems.
+But when *first* packaging existing software with Nix, it's common to encounter errors that seem inscrutable.
 
-In this tutorial, you'll create your first Nix derivations to package C/C++ software, taking advantage of the [Nixpkgs Standard Environment](https://nixos.org/manual/nixpkgs/stable/#part-stdenv) (`stdenv`) which automates much of the work of building self-contained C/C++ packages.
+## Introducton
 
-The tutorial begins by considering `hello`, an implementation of "hello world" which only requires dependencies provided by `stdenv`.
+In this tutorial, you'll create your first [Nix derivations](https://nix.dev/manual/nix/2.18/language/derivations) to package C/C++ software.
+This takes advantage of the [Nixpkgs Standard Environment](https://nixos.org/manual/nixpkgs/stable/#part-stdenv) (`stdenv`), which reliably automates much of the work involved.
 
+### What will you learn?
+
+The tutorial begins with `hello`, an implementation of "hello world" which only requires dependencies already provided by `stdenv`.
 Next, you will build more complex packages with their own dependencies, leading you to use additional derivation features.
 
 You'll encounter and address Nix error messages, build failures, and a host of other issues, developing your iterative debugging techniques along the way.
 
+### What do you need?
+
+- Familiarity with the Unix shell and plain text editors
+- You should be confident with [reading the Nix language](reading-nix-language). Feel free to go back and work through the tutorial first.
+
+### How long does it take?
+
+Going through all the steps carefully will take ca. 60 minutes.
+
+## Your first package
+
 :::{note}
-A _package_ is an informally defined Nixpkgs concept referring to a Nix derivation representing an installation of some project.
+A _package_ is a loosely defined Nixpkgs concept that refers to both a collection of files and other data, or a Nix derivation representing such a collection before it comes into being.
 Packages have mostly standardised attributes and output layouts, allowing them to be discovered in searches and installed into environments alongside other packages.
 
-For the purposes of this tutorial, "package" means something like "result of a derivation"; this is the artifact you or others will use, as a consequence of having "packaged existing software with Nix".
+For the purposes of this tutorial, "package" means something like "result of a derivation".
+This is the artifact you or others will use, as a consequence of having "packaged existing software with Nix".
```

Comment:
```suggestion
This is the artifact you or others will use, as a consequence of having "packaged existing software with Nix", typically by use of the [nix-build](https://nixos.org/manual/nix/stable/command-ref/nix-build) command.
```

Reply:
Thanks for pointing that out. Since I was sort of mindlessly going through the text, I failed to notice a key issue I noticed many people had discussed in the past but not clearly resolved: as the "what's a package note" highlights, we have this tension between final artifact (file system objects) and the artifact that produces the final one (derivation).

This sentence is slightly unclear about what we mean, and your suggestion is to pick one. I suppose we sort of care about both, but probably a bit more about the second here. Because we're writing the derivation code, and the final artifact is made automatically. I'll think about it and make a new suggestion how to phrase it.